designcompose/src/main/java/com/android/designcompose/DocServer.kt

```diff
@@ -218,24 +217,15 @@ internal fun DocServer.initializeLiveUpdate() {
     periodicFetchRunnable =
         Runnable() {
             thread {
-                try {
-                    if (!fetchDocuments(firstFetch)) {
-                        Log.e(TAG, "Error occurred while fetching or decoding documents.")
-                        return@thread
-                    }
-                    firstFetch = false
-                } catch (e: StatusRuntimeException) {
```

Comment:
I don't remember clearly -- did `fetchDocuments()` used to throw a GRPC exception and we just didn't remove this code?

Reply:
Yep, it came from the old FSAAS code. This bit didn't get cleaned up when FSAAS was removed.